src/algokit/cli/init.py

```diff
@@ -181,32 +181,35 @@ def init_command(
         commit=template_url_ref,
         unsafe_security_accept_template_url=unsafe_security_accept_template_url,
     )
+    logger.debug(f"template source = {template}")
 
     project_path = _get_project_path(directory_name)
+    logger.debug(f"project path = {project_path}")
     directory_name = project_path.name
     # provide the directory name as an answer to the template, if not explicitly overridden by user
     answers_dict.setdefault("project_name", directory_name)
 
-    logger.debug(f"Attempting to initialise project in {project_path} from template {template}")
-
+    logger.info("Starting template execution...")
```

Comment:
```suggestion
    logger.info("Starting template initialisation. Initialising in {project_path} from template {template}.")
```

Reply:
So I went with execution over initialization because we use initialization later on, didn't want to confuse things.
Also those values are logged immediately as debug, and the user just provided them, so to keep the output clean and easier to digest I left them out of this message here